terminus-ui/select/src/optgroup/optgroup.component.ts

```diff
@@ -58,7 +55,7 @@ let nextUniqueId = 0;
     role: 'group',
     '[class.ts-optgroup--disabled]': 'isDisabled',
     '[attr.id]': 'id',
-    '[attr.aria-disabled]': 'isDisabled.toString()',
+    '[attr.aria-disabled]': 'isDisabled ? "true" : "false"',
```

Comment:
What was the reason for this change?

Reply:
Once coercion was removed, the tests failed due to "unable to read toString of undefined."

This format is similar to what's used for disabled in ts-input element:
https://github.com/GetTerminus/terminus-ui/blob/master/terminus-ui/input/src/input.component.html#L96